bpfd-api/src/util.rs

```diff
@@ -25,23 +33,15 @@ pub mod directories {
     pub const RTDIR_FS:         &str = "/run/bpfd/fs";
     pub const RTDIR_FS_MAPS:    &str = "/run/bpfd/fs/maps";
     pub const RTDIR_PROGRAMS:   &str = "/run/bpfd/programs";
+
     //
     // CacheDirectory: /var/cache/
+
```

Comment:
Nit: Extra newline

Reply:
I removed some extra new lines. I left newline between different sections. Let me know if you want those gone too, but I think is is more readable this way.